src/APIM_ARMTemplate/apimtemplate/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "apimtemplate": {
       "commandName": "Project",
-      "commandLineArgs": "create --outputLocation=\"C:\\Users\\lucashh\\Desktop\\Projects\\APIM-ARM\\GeneratedTemplates\" --openAPISpecFile=\"C:\\Users\\lucashh\\Desktop\\Projects\\APIM-ARM\\ExampleInputs\\openAPISpecs\\tags.json\" --apiVersionSetFile=\"C:\\Users\\lucashh\\Desktop\\Projects\\APIM-ARM\\ExampleInputs\\apiVersionSets\\valid.yml\""
+      "commandLineArgs": "create --outputLocation=\"C:\\Users\\lucashh\\Desktop\\Projects\\APIM-ARM\\GeneratedTemplates\" --openAPISpecFile=\"C:\\Users\\lucashh\\Desktop\\Projects\\APIM-ARM\\ExampleInputs\\openAPISpecs\\tags.json\" --apiVersionSetFile=\"C:\\Users\\lucashh\\Desktop\\Projects\\APIM-ARM\\ExampleInputs\\apiVersionSets\\valid.yml\" --authenticationSettingsFile=\"C:\\Users\\lucashh\\Desktop\\Projects\\APIM-ARM\\ExampleInputs\\authenticationSettings\\valid.yml\""
```

Comment:
I would think we should have something like this

create --input-file="C:\\Users\\lucashh\\Desktop\\Projects\\APIM-ARM\apimtemplateconfig.yml"

The Yaml would have these settings like

openapispecfile="swagger-petsore.json"
path="petstore"
apiRevision="3"
apiRevisionDescription="Petstore revision having support to list poodles"
apiVersionSet="v1Version"
authenticationSetting